readthedocs/doc_builder/constants.py

```diff
@@ -25,3 +26,16 @@
 DOCKER_OOM_EXIT_CODE = 137
 
 DOCKER_HOSTNAME_MAX_LEN = 64
+
+# Build images
+BUILD_IMAGES = {
+    'readthedocs/build:14.04': {
+        'python': {'supported_versions': [2, 2.7, 3, 3.3]},
+    },
+    'readthedocs/build:16.04': {
+        'python': {'supported_versions': [2, 2.7, 3, 3.5]},
+    },
+    'readthedocs/build:beta': {
```

Comment:
Wonder if this should be given a more specific name? Seems like we'll want to do stuff over time where `beta` isn't really reflective of, and we don't want it to conflict.